src/raids/completeRaid.test.ts

```diff
@@ -59,7 +60,7 @@ it(`does not complete a raid if more than one active raid exists for the dungeon
   // Setup an existing raid
   const raidRepo = buildRepository();
   const raidStats = buildRaidStats({
-    dungeon: String(raidRepo.parent?.full_name),
+    dungeon: raidRepo.parent!.full_name,
```

Comment:
Is this a JS operator I am unaware of?

Reply:
I don't know the name, but is TS only. Basically it just tells the compiler, "This won't be null/undefined, even though you think it will be"